arangod/IResearch/IResearchView.cpp

```diff
@@ -262,6 +263,12 @@ struct IResearchView::ViewFactory : public arangodb::ViewFactory {
       );
     }
 
+    if (meta._version == 0 && inUpgrade) {
+      LOG_TOPIC("deadbeef", FATAL, arangodb::iresearch::TOPIC)
```

Comment:
```suggestion
      LOG_TOPIC("8db62", FATAL, arangodb::iresearch::TOPIC)
```
Log ids must be 5 digits long